config/locales/en.yml

```diff
@@ -461,6 +462,7 @@ en:
   protocols:
     new: "New Protocol"
     edit: "Edit %{protocol_type} Information"
+    edit_study: "Edit Study Information"
```

Comment:
This should be using the `edit` key above and passed a `protocol_type` parameter